src/config.rs

```diff
@@ -32,6 +32,12 @@ impl Root {
     pub fn appenders(&self) -> &[String] {
         &self.appenders
     }
+
+    /// Sets the minimum level of log messages that the root logger will accept.
+    pub fn set_level(&mut self, level: LevelFilter) -> LevelFilter {
```

Comment:
Why does this return the same thing it's passed?

Reply:
This is merely a convention I am used to; if you think it is unnecessary, I can remove it.

Reply:
Let's remove it.

Reply:
Resolved